config/root-config.in

```diff
@@ -48,6 +48,7 @@ libdir=@libdir@
 incdir=@incdir@
 etcdir=@etcdir@
 tutdir=@tutdir@
+all_features="@all_features"
```

Comment:
```suggestion
all_features="@all_features@"
```
? Please make sure this actually works before submitting the PR :-) (Maybe you did, just stating the "obvious")